### PR TITLE
feat(dashboard): add container image

### PR DESCRIPTION
**Pull request by anbraten**
Adds a container image for the dashboard.

# TODO
- [x] fix static file serving

### CODE REVIEW

File: .github/workflows/container.yml [hunk 1]
```diff
@@ -17,8 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        # image: [serverless, dashboard]
-        image: [serverless]
+        image: [serverless, dashboard]
     steps:
       - name: Checkout 🛎️
```
Comment (QuiiBz) [inline .github/workflows/container.yml:22]: Can we remove this emoji btw?

File: docker/Dockerfile.dashboard [hunk 2]
```diff
@@ -0,0 +1,53 @@
+FROM node:18-alpine AS base
```
Comment (QuiiBz) [inline docker/Dockerfile.dashboard:1]: We use `node:20.2.0-bullseye-slim` in `Dockerfile.dashboard`, let's align and not have multiple versions

File: docker/Dockerfile.dashboard [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+RUN adduser --system --uid 1001 nextjs
+
+COPY --from=builder /app/packages/dashboard/public ./public
+
+# Automatically leverage output traces to reduce image size
+# https://nextjs.org/docs/advanced-features/output-file-tracing
+COPY --from=builder --chown=nextjs:nodejs /app/packages/dashboard/.next/standalone ./
+COPY --from=builder --chown=nextjs:nodejs /app/packages/dashboard/.next/static ./packages/dashboard/.next/static
+
+RUN ls -la .
```
Comment (QuiiBz) [inline docker/Dockerfile.dashboard:45]: Do we need this `ls`?

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks! Some feedback:

File: docker/Dockerfile.dashboard (hunk 3 above)
Comment (anbraten) [reply docker/Dockerfile.dashboard]: I just used it for debugging to see if the files were build and copied correctly

File: docker/Dockerfile.dashboard (hunk 2 above)
Comment (anbraten) [reply docker/Dockerfile.dashboard]: sure

File: .github/workflows/container.yml (hunk 1 above)
Comment (anbraten) [reply .github/workflows/container.yml]: removed it

**Comment by QuiiBz** [comment]
It fails to build due to Sentry, not sure why 🤔 

https://github.com/lagonapp/lagon/actions/runs/5168297080/jobs/9309789857?pr=906#step:8:538

**Review by QuiiBz — APPROVED** [review]
Thanks! I've fixed the Next.js build errors and it's now working as expected

**Pull request merged**